packages/pharos/src/components/image-card/pharos-image-card.scss

```diff
@@ -165,7 +165,28 @@ slot[name='image']::slotted(img) {
 .card__link--collection {
   display: grid;
   position: relative;
-  height: auto;
+  height: 14rem;
```

Comment:
Updated the `height` to match the `height` set with image link and container error. Setting the height also fixes the mismatched heights when its set to `auto` https://github.com/ithaka/pharos/blob/develop/packages/pharos/src/components/image-card/pharos-image-card.scss#L23

Reply:
@NazimHAli it seems like this potentially impacts visual/editorial design fairly significantly, comparing how much of the image covers the card area. Is there a sense from design whether this is alright?

Reply:
I noticed this too while reviewing collection variant in prod vs this, testing it with the same height here https://github.com/ithaka/pharos/pull/516/commits/5000badd5f55110a1d49d7ebb19f89aca6fee28a